src/components/Todo/Todo.js

```diff
@@ -1,8 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ListItem from 'components/ListItem/ListItem'
+import Input from 'components/Input/Input'
+import Button from 'components/Button/Button'
+import { FaEdit } from 'react-icons/fa'
+import { MdCancel } from 'react-icons/md'
 
-const Todo = ({ children, ...props }) => {
-  return <ListItem {...props}>{children}</ListItem>
+const Todo = ({ onUpdateText, id, value, children, ...props }) => {
+  const [editing, setEditing] = useState(false)
+  const [toDoText, setToDoText] = useState(value)
```

Comment:
El `value` no es necesario, podés usar `children` que contiene el texto del todo. Al igual que en el comentario de arriba, `todo` no va en `camelCase`

```suggestion
  const [todoText, setTodoText] = useState(children)
```